docs/Changelog.md

```diff
@@ -29300,6 +29365,133 @@ This version of the operator has been available since version 23 of the default
 <dd>Constrain output to int64 tensor, which should be a scalar though.</dd>
 </dl>
 
+### <a name="SkipLayerNormalization-23"></a>**SkipLayerNormalization-23**</a>
```

Comment:
Is there enough motivation to add SkipLayerNormalization and SkipRMSNormalization into the spec if they can be simply expressed using Mul, Add and existing Normalization ops?